sprints/20201102-object-service/sprint-retrospective.md

```diff
@@ -0,0 +1,50 @@
+* Team load sharing
+  * Once again the team proved flexible and adaptable in shifting between disparate epics and going where the need was the greatest.
+
+## What didn't
+* Lack of engagement for sprints when not driving them
+  * Direct ownership of functional areas is appealing, but is it better? As noted above, not having single points of failure or silos of knowledge is important, especially from a business perspective.
+  * There isn't always enough "interesting" work to go around every sprint. By design, the Product owner should feel OK (or even obligated) to pick up the bigger, more interesting pieces of work. This can be seen as a benefit to taking the time to organize and plan the sprint.
+  * *However*, the counterpoint is that the remaining work is less palatable to the rest of the team. This team has many senior engineers, and picking up the scraps is less appealing.
+* Planning the work
+  * Unambiguously, proper planning of the sprints is important and leads to better outcomes.
+  * As we saw in the postgres sprint, the planning process can/should involve code. This can take the form of tooling to facilitate team involvement and output consistency, or testing/metrics frameworks to validate and measure outcomes.
+* Sprints work well once we achieve a certain level of maturity
+  * During early work/design, a more singular focus is required. The Product owner is affored this luxury of time.
+  * The Product owner can spend that time to solidify design decisions (and explore alternatives) that would be challenging to revisit mid-sprint.
+  * Caveat: this doesn't mean we will always find the correct path at the outset. Team members must still be free to propose better alternatives to avoid succumbing to the sunk cost fallacy.
+* Do sprints need to involve everyone?
+  * Short answer: No
+    * In the future, we should aim to have shorter sprints for fewer people.
+    * Sprints should only involve as many people as required. People who aren’t involved can be handling interrupts, high-priority fixes, or planning future sprints.
+    * This would result in more targeted sprints. We should avoid adding "filler" epics to sprints just to involve everyone.
+  * This would also make working around PTO easier.
+  * Caveat: we need to make sure people rotate through sprint involvement to make sure no one is stuck handling interrupts all the time. Avoid "interrupt duty."
+* Should we have longer slush weeks?
+  * Yes and no
+  * Having more time for investigations would be good, but if sprints are more targeted and don't necessarily involve the whole team, those team members not involved in the sprint will have more opportunity to do this. We recognize that the time for digressions and explorations based on discovery is important.
+  * “Slush week” needs a better name
+    * Proposals:
+      * Individual exploration time
+      * Jogs
```

Comment:
this one feels like work is being done just at a different pace. maybe that's good?